File: Roll No.33_Practice_Problems/33_Practice_Problem_2.cpp
#include<iostream>
#include<string>
using namespace std;
class Employee
{
	int emp[100];
	int emp_num;
	string name;
	float Basic;
	float DA;
	float IT;
	float Net_Salary;
	public:
		void get_data(){
		cout<<"Enter Employee name "<<endl;
		cin>>name;
		cout<<"Enter Employee ID Number "<<endl;
		cin>>emp_num;
		cout<<"Enter Employee Basic salary "<<endl;
		cin>>Basic;
		}
		void calc();
		void print_data();
};
void Employee::calc()
{
	DA=(1.32)*(Basic);
	IT=(0.30)*(Basic+DA);
	Net_Salary= (Basic+DA)-(IT);
}
void Employee::print_data()
{
		cout<<"\nThe name of Employee is "<<name<<endl;
		cout<<"The ID Number of Employee is "<<emp_num<<endl;
		cout<<"The Basic Salary of Employee is "<<Basic<<endl;
		cout<<"The net salary of the Employee is "<<Net_Salary<<endl<<endl;
}
int main()
{
	Employee obj;
	for(int i=0;i<3;i++)
	{
		obj.get_data();
		obj.calc();
		obj.print_data();
	}
	return 0;
}

File: Roll No.33_Practice_Problems/33_Practice_Problem_2.java
import java.util.Scanner;

public class Employee {
	Scanner sc=new Scanner(System.in);
	String name;
	int emp_num;
	float Basic;
	float DA;
	float IT;
	float Net_Salary;
	void get_data()
	{
		System.out.println("Enter Employee name ");
	    name=sc.next();
		System.out.println("Enter Employee ID Number ");
		emp_num=sc.nextInt();
		System.out.println("Enter Employee Basic Salary ");
		Basic=sc.nextFloat();
	}
	void calc()
	{
		DA=(float) (1.32)*(Basic);
		IT=(float) (0.30)*(Basic+DA);
		Net_Salary=(float) (Basic+DA)-(IT);
	}
	void display() {
		System.out.println("The Employee name is "+name);
		System.out.println("The Employee ID Number is "+emp_num);
		System.out.println("The Employee Basic Salary is "+Basic);
		System.out.println("The Employee Net Salary is "+Net_Salary);
};
	public static void main(String[] args) {
	Employee obj=new Employee();
	for(int i=0;i<3;i++) {
		obj.get_data();
		obj.calc();
		obj.display();
	}
	

	}

}
